### DISCUSSION TITLE
I cannot create a new repository

### DISCUSSION THREAD

**Discussion by wintan1418** [community/community#27453] [Other Feature Feedback, Questions, & Ideas]
Hy guys, please I am trying to create a new repository but the create button is disabled, I am wondering if there is a limit to the amount of repo I can have cus currently I have 100 repo thereabout, Please I would appreciate an urgent help, Thanks.

**Comment by Aravindrajeev432** [comment]
Please read this:

https://stackoverflow.com/questions/67565130/create-repository-button-disabled-on-github

**Comment by Aravindrajeev432** [comment]
everyone can has an unlimited number of public and private repositories even as part of a free plan

**Reply by wintan1418** [reply]
thanks @Aravindrajeev432 , but when I do the same it still stays disabled

**Comment by jfullstackdev** [comment]
can you send screenshots of those, so we can take a closer look, 

as usual, extensions can affect GitHub, so if you installed GitHub extensions, they may affect it

second, try logging in in Incognito mode and see whether it will work there, Incognito is the fresh copy of your tab

**Reply by wintan1418** [reply]
![Uploading github.png…]()

**Reply by jfullstackdev** [reply]
is this done ? what I see is this 

![image](https://user-images.githubusercontent.com/47092464/181508791-b512c8ac-3206-41ed-9285-dfaf47bd35af.png)

**Reply by wintan1418** [reply]
![github](https://user-images.githubusercontent.com/60691122/181509291-6a60be8a-f050-4ab5-8cb0-8e95baf8e614.png)

**Reply by jfullstackdev** [reply]
why, do you have Orgs ? you must select an owner like this

![image](https://user-images.githubusercontent.com/47092464/181512923-3fdfa435-e416-4552-b21a-4939e0a71a1a.png)

**Reply by wintan1418** [reply]
Thanks man, that did it, funny  how I never noticed
